pom.xml

```diff
@@ -831,6 +831,17 @@
         </resources>
     </build>
     <profiles>
+        <profile>
+            <activation>
+                <property>
+                    <name>build.snapshot</name>
+                    <value>!false</value>
+                </property>
+            </activation>
+            <properties>
+                <version>1.1.0.0-SNAPSHOT</version>
```

Comment:
Looks like this PR is bumping security version from `1.0.1.0` to `1.1.0.0`. If that's the case, is that the case?

Reply:
Yes, this is the case.

Reply:
Must be consistent with `plugin-descriptor.properties`, I don't see how this can be done using profiles.

Reply:
Update: by default the version comes from plugin.properties and is set to 1.1.0.0. This sets the version to 1.1.0.0-SNAPSHOT unless you specify `-Dbuild.snapshot=false`. Meaning it sets it by default. This effectively enables building both, snapshot or not snapshot, with snapshot by default.

If you put 1.1.0.0-SNAPSHOT in the properties file, then this profile condition would have to be inverted. But then the default can't be snapshot=true.